src/processes.c

```diff
@@ -1411,6 +1478,12 @@ static int ps_read_process(long pid, process_entry_t *ps, char *state) {
     return 0;
   }
 
+  if (ps_get_pss(ps) != 0) {
+    /* no pss data */
+    ps->vmem_pss = -1;
```

Comment:
`vmem_pss` is an `unsigned long`. Assigning -1 here will cause problems with compilers. I suggest to use zero here instead.

Reply:
I first thought there could be some tasks with zero PSS, but kernel tasks apparently do not have `smaps_rollup` files, and their `smaps` files are empty, so zero PSS seems OK indicator for missing data (otherwise I would have suggested `ULONG_MAX`).